databricks/sdk/runtime/__init__.py

```diff
@@ -16,6 +16,18 @@
 except ImportError:
     init_runtime_native_auth = None
 
+if init_runtime_native_auth is None:
+    try:
+        from dbruntime.databricks_repl_context import get_context
+        ctx = get_context()
```

Comment:
Shall we try reading the env variable `DATABRICKS_TOKEN` as well. We put it in multiple documentations.
https://docs.databricks.com/dev-tools/python-sql-connector.html
https://python.langchain.com/en/latest/integrations/databricks.html

Reply:
it's handled generically in the earlier stage of config resolution: https://github.com/databricks/databricks-sdk-py/blob/main/databricks/sdk/core.py#L704-L717